relay-quotas/src/is_rate_limited.lua

```diff
@@ -17,7 +17,7 @@
 -- Send these values:
 --
 --     KEYS = {"foo", "foo_refund", "bar", "bar_refund"}
---     ARGV = {10, 600 + now(), 5, 20, 600 + now(), 1}
+--     ARGV = {10, 600 + now(), 5, false, 20, 600 + now(), 1, true}
```

Comment:
Let's also add the new arg to the description starting on line 8.

Reply:
done in [6875902](https://github.com/getsentry/relay/pull/1521/commits/6875902fb62f52263758c3711f8e898777003bcc)